src/components/VTable/VTable.vue

```diff
@@ -16,123 +16,127 @@
             )}
         </dl>
       </div>
-    </div> -->
+    </div>
+  -->
 
-    <div
-      ref="container"
-      class="table-container"
-      tabindex="0"
-      role="group"
-      aria-labelledby="caption"
-    >
-      <table>
-        <caption v-if="caption" id="caption">
-          {{ caption }}
-        </caption>
-        <thead v-if="headers.length">
-          <tr>
-            <th
-              v-for="(header, key) in cHeaders"
-              :key="key"
-              role="columnheader"
-              :aria-sort="
-                sortBy !== header.key
-                  ? null
-                  : sortOrder === 'ASC'
-                  ? 'ascending'
-                  : 'descending'
-              "
-            >
-              {{ header.text || header.key }}
+  <div
+    ref="container"
+    class="table-container"
+    tabindex="0"
+    role="group"
+    aria-labelledby="caption"
+  >
+    <table>
+      <caption
+        v-if="caption"
+        id="caption"
+      >
+        {{ caption }}
+      </caption>
+      <thead v-if="headers.length">
+        <tr>
+          <th
+            v-for="(header, key) in cHeaders"
+            :key="key"
+            :aria-sort="ariaSort(header)"
+          >
+            {{ header.text || header.key }}
 
-              <button
-                v-if="header.sortable"
-                :aria-label="
-                  `sort by ${header.text || header.key} in ${
-                    !sortOrder
-                      ? 'ascending'
-                      : sortOrder === 'ASC'
-                      ? 'descending'
-                      : 'default'
-                  } order`
-                "
-                @click="header.sortable && onSort(header.key)"
+            <!-- eslint-disable vue-a11y/click-events-have-key-events -->
+            <button
+              v-if="header.sortable"
+              :aria-label="ariaLabel(header)"
+              @click="onSort(header.key)"
```

Comment:
changed from `header.sortable && onSort(header.key)` to just `onSort(header.key)` because the `v-if` should handle the rest.